test/acceptance/pages/TestOnlyReportProblemPartialPage.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.pages
+
+import acceptance.support.Helpers
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.{ExpectedConditions, WebDriverWait}
+
+object TestOnlyReportProblemPartialPage extends TestOnlyReportProblemForm {
+  val url: String = wrapUrl("/test-only")
+}
+
+object TestOnlyReportProblemPartialPageAjax extends TestOnlyReportProblemForm {
+  val url: String = wrapUrl("/test-only/ajax")
+}
+
+trait TestOnlyReportProblemForm extends BasePage with Helpers {
+
+  val pageTitle: String      = "Service test page – GOV.UK"
+  val welshPageTitle: String = "Service test page – GOV.UK"
+  val expectedSubHeading     = "Thank you"
```

Comment:
@JoPintoPaul I'd be tempted to call this something like, when I first read the assertion below and had to scan back up to double check the intent

```suggestion
  val headingExpectedAfterSuccessfullySending = "Thank you"
```